### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by isentropic**
direct to the working link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,6 @@
 ![compat](https://github.com/FluxML/DataAugmentation.jl/actions/workflows/CompatHelper/badge.svg)
 
 
-[Documentation | Latest](https://fluxml.github.io/DataAugmentation.jl/dev/documents/README.md)
+[Documentation | Latest]([https://fluxml.github.io/DataAugmentation.jl/dev/documents/README.md](https://fluxml.ai/DataAugmentation.jl/dev/))
```
Comment (adrhill) [inline README.md]: I don't think we need to keep the broken URL around:
```suggestion
[Documentation | Latest](https://fluxml.ai/DataAugmentation.jl/dev/)
```

File: README.md [hunk 2]
```diff
@@ -4,6 +4,6 @@
 ![compat](https://github.com/FluxML/DataAugmentation.jl/actions/workflows/CompatHelper/badge.svg)
```
Comment (adrhill) [inline README.md]: While we're at it, we could also fix the badges:

```suggestion
![compat](https://github.com/FluxML/DataAugmentation.jl/actions/workflows/CompatHelper.yml/badge.svg)
```

and in the line above:
```diff
- ![CI](https://github.com/FluxML/DataAugmentation.jl/actions/workflows/CI/badge.svg)
+ ![CI](https://github.com/FluxML/DataAugmentation.jl/actions/workflows/ci.yml/badge.svg)
```

**Review by adrhill — CHANGES_REQUESTED** [review]

**Comment by isentropic** [comment]
right

**Review by CarloLucibello — APPROVED** [review]

**Review by adrhill — APPROVED** [review]

**Comment by adrhill** [comment]
Thanks! :)

**Pull request merged**